lfs/gitscanner_catfilebatch.go

```diff
@@ -136,31 +107,25 @@ func (s *CatFileBatchScanner) Scan(sha string) bool {
 		return false
 	}
 
+	fmt.Println("returning true with blobsha:", b, "contents:", c, err)
```

Comment:
Did you mean to leave this here? Should probably use `tracerx.Printf()` for any debug messages that you want to leave in...